desktop/core/src/desktop/auth/api_authentications.py

```diff
@@ -125,6 +127,30 @@ def _handle_public_key(self, access_token):
 
             return public_key_pem
 
+  def _handle_jku_ha(self):
+    res = None
+
+    key_server_urls = AUTH.JWT.KEY_SERVER_URL.get()
+    if not key_server_urls:
+      return None
+
+    if "," in key_server_urls:
+      # Config format is "['url1', 'url2']" for HA, so we need to clean up and split correctly in list.
+      key_server_urls_list = key_server_urls.replace("%20", "").replace("['", "").replace("']", "").replace("'", "").split(',')
```

Comment:
this seems like a json array. Why not just do a json deserialize?

Reply:
Looked a bit more on this and turns out we don’t need json deserialize too.

Earlier I was using the changes done like this: https://github.com/cloudera/hue/blob/88905cc4ecef89cad518d763d732e1ae520eaca6/desktop/libs/metadata/src/metadata/catalog/atlas_client.py#L71-L72

We can actually use `type=coerce_string`  instead of `type=str` in the config, and then we get a nicely formatted config which we can do a simple `.split(',')`

I’ll change all implementations now where this `.replace()` stuff is there.